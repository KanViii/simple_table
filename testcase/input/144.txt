INSERT Abc123_ number