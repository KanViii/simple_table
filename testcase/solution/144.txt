Invalid: ASSIGN y 'abbc@@1'