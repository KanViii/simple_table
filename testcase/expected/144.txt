Invalid: INSERT Abc123_ number